### ISSUE TITLE
cell state label is mandatory ? 

### ISSUE THREAD

**Issue by martina811** [Danko-Lab/BayesPrism#66]
Hi, I would like to use BayesPrism on rnaseq data, is it mandatory to have also a file which specify the cell state label of my single cell data? Or can it work with just the cell type label ?  

many thanks

**Comment by tinyi** [comment]
Yes. To only use cell type, simply specify cell.state.labels = the same
input you provided for cell.type.labels.

Theoretically speaking, the definition of cell states were developed to
account for the uncertainty in the reference, particularly for cell type
whose distribution is heterogeneous and lies on a continuum,  and hence was
not developed for accurate quantification. The reason for this is that
suppose cell state B is a linear combination of cell state A and C, even
though they are presented in the same amount in the mixture, the inference
can be unstable (prone to the data quality of the cells that constitute the
cell state), and the sparse prior may also penalize cellstate B to close to
0.

**Comment by yunbokai** [comment]
Hi @tinyi , I want to estimate and compare the fraction of Tcell_sub1, Tcell_sub2 and Tcell_sub3 in different RNA-seq data. Is it reasonable to set Endothelial, Fibroblats, Epi, Myeloids, Tcell_sub1, Tcell_sub2 and Tcell_sub3 as cell type label and set Endothelial_sub1, Endothelial_sub2, Fibroblats_sub1, Fibroblats_sub2, Epi_sub1, Epi_sub2, Myeloids_sub1, Myeloids_sub2, Tcell_sub1, Tcell_sub2 and Tcell_sub3 as cell state label? Or do you have any other suggestion? Thanks.
